packages/fungi/fungi.0.1.4/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "A pure functional graph library"
+description: """\
+A graph library based on functional adjacency list with some common graph
+algorithms implemented and a fibonacci heap."""
+maintainer: "Harry K <kituyiharry@github.com>"
+license: "MIT"
```

Comment:
Several required fields are missing here:

```
Error in fungi.0.1.4: warning 37: Missing field 'dev-repo'
Error in fungi.0.1.4: warning 36: Missing field 'bug-reports'
Error in fungi.0.1.4: warning 35: Missing field 'homepage'
Error in fungi.0.1.4: warning 25: Missing field 'authors'
```

These are reported in the CI results: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/329441876953d310d81578603583aa15a960b1d5